code/modules/paperwork/paperbin.dm

```diff
@@ -103,14 +123,18 @@
 	else
 		return ..()
 
+/obj/item/paper_bin/proc/at_overlay_limit()
+	if(overlays.len >= MAX_ATOM_OVERLAYS)
+		return TRUE
+	return FALSE
+
```

Comment:
You can just return the check wrapped in () if you'd like

Reply:
Thanks